with-firebase-saml-login/App.js

```diff
@@ -0,0 +1,126 @@
+// Import third-party dependencies
+import React from 'react'
+import { View, Button, LogBox, StyleSheet, Text } from 'react-native'
+import * as WebBrowser from 'expo-web-browser';
+import * as Linking from 'expo-linking';
+
+import firebase from 'firebase'
+
+const firebaseConfig = {// !!!!!PUT YOUR FIREBASE CONFIG HERE!!!!!
+  apiKey: "AIzaSyAlZruO2T_JNOWn4ysfX6AryR6Dzm_VVaA",
+  authDomain: "blobtest-36ff6.firebaseapp.com"
+};
```

Comment:
```suggestion
// Add Firebase configuration here: [LINK TO GET ONE]
const firebaseConfig = {
  // apiKey: "XXXX",
  // authDomain: "XXXX.firebaseapp.com"
};
```

Be sure to add info to the README with instructions on how the user can setup and run the project